tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -589,7 +606,7 @@ def docomplicated(suptitle=None):
 # much smaller than the left-hand, so the right-hand layoutboxes are smaller.
 #
 # The Subplotspec boxes are laid out in the code in the subroutine
-# `.arange_subplotspecs`, which simply checks the subplotspecs in the code
+# ``.arange_subplotspecs``, which simply checks the subplotspecs in the code
```

Comment:
Either no leading dot (and possibly trailing parentheses) or a proper link.

Reply:
This is the method or function that does not even exists. Maybe @jklymak can comment here and in the cases below how this is to be understood and how those names are in general to be layouted. Maybe there is a deeper sense in the leading-dot-notation here?

Reply:
I assume that it was thought to be a link. `` `.arange_subplotspecs` `` would be perfectly fine if the module wasn't private.

Reply:
Oh stupid, of course you're right. Will change that.